trackpy/feature.py

```diff
@@ -486,14 +492,22 @@ def locate(raw_image, diameter, minmass=None, maxsize=None, separation=None,
     .. [1] Crocker, J.C., Grier, D.G. http://dx.doi.org/10.1006/jcis.1996.0217
 
     """
+    if invert:
+        warnings.warn("The invert argument will be deprecated. Use a PIMS "
+                      "pipeline for this.", DeprecationWarning)
+    if filter_before is not None:
```

Comment:
I think this should raise if `filter_before` is True, or at least rephrase the warning. It says the feature "will be deprecated," which I read as meaning "the filtering is still being done now, but it won't be in a future release."